packages/shared-ux/page/kibana_no_data/impl/src/services.tsx

```diff
@@ -47,10 +47,11 @@ export const KibanaNoDataPageKibanaProvider: FC<KibanaNoDataPageKibanaDependenci
   children,
   ...dependencies
 }) => {
-  const { dataViews } = dependencies;
+  const { dataViews, coreStart } = dependencies;
   const value: Services = {
     hasESData: dataViews.hasData.hasESData,
     hasUserDataView: dataViews.hasData.hasUserDataView,
+    hasCustomBranding: coreStart.customBranding.hasCustomBranding,
```

Comment:
We don't want the entire `coreStart` as a dependency here; that was the whole point of writing our own service wrapper. You will need to provide `customBranding` FROM `coreStart` in the consuming plugins.